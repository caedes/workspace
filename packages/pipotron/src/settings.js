import adjectives from './data/adjectives';
import adverbs from './data/adverbs';
import infinitives from './data/infinitives';
import names from './data/names';
import prepositions from './data/prepositions';
import pronouns from './data/pronouns';
import solutionAdjectives from './data/solution-adjectives';
import solutionNames from './data/solution-names';
import verbs from './data/verbs';

export default [
  prepositions,
  names,
  adjectives,
  ',',
  verbs,
  infinitives,
  pronouns,
  solutionNames,
  solutionAdjectives,
  adverbs,
  '.',
];
