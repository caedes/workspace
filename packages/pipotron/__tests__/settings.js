import settings from '../src/settings';

describe('settings', () => {
  it('returns the settings', () => {
    expect(settings).toMatchSnapshot();
  });
});
