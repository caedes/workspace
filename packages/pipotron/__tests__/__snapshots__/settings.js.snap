// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`settings returns the settings 1`] = `
Array [
  Array [
    "Avec",
    "Considérant",
    "Où que nous mène",
    "Eu égard à",
    "Vu",
    "En ce qui concerne",
    "Dans le cas particulier de",
    "Quelle que soit",
    "Du fait de",
    "Tant que durera",
    "Quoi qu'on dise concernant",
    "Nonobstant",
    "Compte tenu de",
    "Malgré",
    "Pour réagir face à",
    "Afin de circonvenir à",
    "Dans le but de pallier à",
    "Si vous voulez mon avis concernant",
  ],
  Array [
    "la restriction",
    "l'orientation",
    "la crise",
    "l'inertie",
    "la difficulté",
    "l'austérité",
    "la dégradation",
    "cette rigueur",
    "la dualité de la situation",
    "la baisse de confiance",
    "la morosité",
    "la situation",
    "l'ambiance",
    "la politique",
    "la fragilité",
    "complexité",
    "l'inconstance",
    "cette inflexion",
  ],
  Array [
    "présente",
    "actuelle",
    "générale",
    "induite",
    "conjoncturelle",
    "observée",
    "contextuelle",
    "de ces derniers temps",
    "de l'époque actuelle",
    "intrinsèque",
    "que nous constatons",
  ],
  ",",
  Array [
    "il convient d#",
    "il faut",
    "on se doit d#",
    "il est préférable d#",
    "il serait intéressant d#",
    "il ne faut pas négliger d#",
    "on ne peut se passer d#",
    "il est nécessaire d#",
    "il serait bon d#",
    "il faut de toute urgence",
    "je recommande d#",
    "je préconise un audit afin d#",
    "il est très important d#",
    "il ne faut pas s'interdire d#",
    "nous sommes contraints d#",
    "je suggère fortement d#",
    "je n'exclus pas d#",
    "je vous demande d#",
  ],
  Array [
    "étudier",
    "examiner",
    "favoriser",
    "prendre en considération",
    "anticiper",
    "imaginer",
    "uniformiser",
    "remodeler",
    "avoir à l'esprit",
    "se remémorer",
    "gérer",
    "fédérer",
    "comprendre",
    "analyser",
    "appréhender",
    "expérimenter",
    "essayer",
    "caractériser",
    "façonner",
    "revoir",
    "prendre en compte",
    "arrêter de stigmatiser",
    "considérer",
    "réorganiser",
    "inventorier",
  ],
  Array [
    "toutes les",
    "chacune des",
    "la majorité des",
    "la simultanéité des",
    "l'ensemble des",
    "la somme des",
    "la totalité des",
    "la globalité des",
    "les relations des",
    "certaines",
    "la plus grande partie des",
    "les principales",
    "systématiquement les",
    "précisément les",
  ],
  Array [
    "solutions",
    "issues",
    "problématiques",
    "voies",
    "alternatives",
    "organisations matricielles",
    "améliorations",
    "ouvertures",
    "synergies",
    "actions",
    "options",
    "décisions",
    "modalités",
    "hypothèses",
    "stratégies",
  ],
  Array [
    "imaginables",
    "possibles",
    "s'offrant à nous",
    "de bon sens",
    "envisageables",
    "éventuelles",
    "réalisables",
    "déclinables",
    "pertinentes",
    "que nous connaissons",
    "évidentes",
    "optimales",
    "opportunes",
    "emblématiques",
    "draconiennes",
  ],
  Array [
    "à long terme",
    "pour longtemps",
    "à l'avenir",
    "pour le futur",
    "depuis longtemps",
    "à court terme",
    "rapidement",
    "dans une perspective correcte",
    "avec toute la prudence requise",
    "de toute urgence",
    "même si ce n'est pas facile",
    "même si nous devons en tirer des conséquences",
    "très attentativement",
    "avec beaucoup de recul",
    "parce que la nature a horreur du vide",
    "parce que nous ne faisons plus le même métier",
    "toutes choses étant égales par ailleurs",
    "et déjà en notre possession",
    "en prenant toutes les précautions qui s'imposent",
    "si l'on veut s'en sortir un jour",
    "parce qu'il est temps d'agir",
    "parce qu'il s'agit de notre dernière chance",
    "parce que les mêmes causes produisent les mêmes effets",
    "parce que nous le valons bien",
  ],
  ".",
]
`;
